### ISSUE TITLE
Unable to write logs to disk

### ISSUE THREAD

**Issue by jbarbadillo** [orhanobut/logger#180]
I have tried the following configuration with version 2.2.0:

```Java
FormatStrategy csvFormatStrategy = CsvFormatStrategy.newBuilder()
                .tag("custom")
                .build();

Logger.addLogAdapter(new DiskLogAdapter(csvFormatStrategy));

Logger.i( "onCreate");
```
I am unable to find the logs. I am looking for them in 
sdcard/Android/data/myappname

The documentation on readme.md does not specify anything else. Am I missing something?

**Comment by orhanobut** [comment]
I agree that documentation about this particular `LogAdapter` is not good enough. As default it writes to the external storage. It also requires some permissions. Do you have appropriate permissions to write to a file?

**Comment by jbarbadillo** [comment]
Yes I have external storage permissions.

I tried to look for the log in the system folder using adb shell but I could not find it anywhere.

I managed to solve this issue by manually checking if directory exists and creating it if not, as I saw on Issue #143. I also implemented the **DiskLogHandler** shown on that thread to use a different storage location.

**Comment by RyPoints** [comment]
So, I just got this functional and after reading this was looking in sdcard/Android/data/myappname.  The default save location is not sdcard/Android/data/myappname.  The default save location is:

`SD Card/logger/logs_0.csv`

Like @orhanobut said you may also need to add:

` <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />`

to your AndroidManifest.xml and you may also need to click enable Apps & Notifications ➡️ App Name ➡️ Permissions ➡️ Storage in the Settings settings for your app.
